integration/integrationAccuracyExperiment.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Thoughts about integration (for error evalutaion of numerical solutions).
+
+Created on Mon Dec 19 13:21:36 2022
+
+@author: maxvondanwitz
+"""
+import numpy as np
+from sympy import symbols, integrate, sqrt
+from sympy.plotting import plot3d
+from sympy import lambdify
+
+
+def monteCarloIntegration(nn, runs):
+    from numpy import linalg as la
```

Comment:
I would not do importing libraries in a function, instead, I import them at the beginning always. But as you wish.

Reply:
Thanks for the hint. I adjusted the code accordingly.